plugin/src/main/java/org/jenkinsci/plugins/structs/describable/DescribableModel.java

```diff
@@ -36,17 +36,7 @@
 import java.lang.reflect.Modifier;
 import java.lang.reflect.Type;
 import java.net.URL;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.LinkedHashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
-import java.util.Stack;
-import java.util.TreeMap;
+import java.util.*;
```

Comment:
FWIW I recommend explicit imports (in a simple lexicographic sort). Less chance of accidental ambiguity. Humans are not supposed to be reading `import` blocks anyway—it is between the IDE and the compiler. Of course we have no mechanically applied code style in most Jenkins repositories so you can do whatever you like so long as you deal with the merge conflicts later and keep diffs minimal.

Reply:
yeah, IDEA tends to do that to my code automatically and I'm always struggling to undo that.  I need to see if there is a way to stop it from doing that.